.github/workflows/rust.yml

```diff
@@ -91,13 +91,6 @@ jobs:
                   sparse-checkout: 'rust/'
                   sparse-checkout-cone-mode: false
 
-            - name: Login to DockerHub
-              if: needs.changes.outputs.rust == 'true'
-              uses: docker/login-action@v2
-              with:
-                  username: posthog
-                  password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
To fix the issue while still addressing @xvello rate limit consideration, let's keep the step, but with [`continue-on-error: true`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error)